packages/kernel-6.1/1006-blk-cgroup-Return-ENOMEM-directly-in-blkcg_css_alloc.patch

```diff
@@ -0,0 +1,68 @@
+From e7e8002397acd1465ed9a6bc7dd8e8eb7537f1f8 Mon Sep 17 00:00:00 2001
```

Comment:
nit: it can help to `git format-patch` the original upstream commit so that this is a useful identifier. This patch is `b5a9adcbd5dc95d34d1f5fc84eff9af6fc60d284` upstream.

Reply:
Makes sense. Will do.